File: ios/MobilePassSDK/Model/DeviceInRange.swift
//
//  DeviceInRange.swift
//  MobilePassSDK
//
//  Created by Erinc Cakir on 28.02.2021.
//

import Foundation
import CoreBluetooth

struct DeviceInRange {
    var serviceUUID: String
    var device:      CBPeripheral
    
    init(serviceUUID: String, bluetoothDevice: CBPeripheral) {
        self.serviceUUID    = serviceUUID
        self.device         = bluetoothDevice
    }
}

File: android/mobilepasssdk/src/main/java/com/armongate/mobilepasssdk/model/DeviceInRange.java
package com.armongate.mobilepasssdk.model;

import android.bluetooth.BluetoothDevice;

public class DeviceInRange {

    public String           serviceUUID;
    public BluetoothDevice  device;

    public DeviceInRange(String serviceUUID, BluetoothDevice bluetoothDevice) {
        this.serviceUUID    = serviceUUID;
        this.device         = bluetoothDevice;
    }

}
